pre_commit/xargs.py

```diff
@@ -106,6 +106,7 @@ def _thread_mapper(maxsize):
 def xargs(cmd, varargs, **kwargs):
     """A simplified implementation of xargs.
 
+    color: Make a TTY
```

Comment:
technically it's a `pty`

Reply:
(and only on platforms which support it)

Reply:
tru

Reply:
updated